homework13_custom_iterator/task_create_iterator.py

```diff
@@ -0,0 +1,30 @@
+"""
+Create an iterator that accepts a sequence and can iterate over values in a given range.
+CustomIterator(sequence, start_index, end_index)
+"""
+
+
+class CustomIterator:
+    def __init__(self, sequence: list, start_index: int, end_index: int):
+        self.__sequence = sequence
+        self.start_index = start_index
+        self.end_index = end_index
+        self.__current_index = self.start_index
+
+    def __iter__(self):
+        return self
+
+    def __next__(self):
+        if self.__current_index in range(self.start_index, self.end_index + 1):
```

Comment:
please think about different use cases.

start index is -10000 and end index is correct
start and end index are negative
and so on. It should work with different sets of data

Reply:
not sure that i understood your question, 
start index - 1000 will fail by default with python index error

''start and end index are negative" , by default python accepts negative indexes, what should i do here?

Reply:
Okay, lets is be -1 index. It will not failed it will start iterating from last element than go to 0 element and so on 

start and end index are negative - yes, but take a look to your implementation. For Example:
start -1 end -4  Data [1,2,3,4,5]
the result should be 5 4 3 2
but I will get 5 1 2 3 4 5 error